### ISSUE TITLE
Unresolved Dependency on Import in Build.SBT (Scala/Play 2.7)

### ISSUE THREAD

**Issue by patientplatypus** [ruippeixotog/scala-scraper#88]
Added `"net.ruippeixotog" %% "scala-scraper" % "2.1.0",` in my `libraryDependencies` and I get:

`sbt.librarymanagement.ResolveException: unresolved dependency: net.ruippeixotog#scala-scraper_2.13;2.1.0: not found
`

So I guess this doesn't work/isn't maintained any more?

**Comment by patientplatypus** [comment]
This site `https://mvnrepository.com/artifact/net.ruippeixotog/scala-scraper-config_2.12/2.1.0` has the config as being slightly different: 

```
// https://mvnrepository.com/artifact/net.ruippeixotog/scala-scraper-config
libraryDependencies += "net.ruippeixotog" %% "scala-scraper-config" % "2.1.0"
```
I get the same errors for both, with the caveat that the name changes between the two versions. Posted my error output below: 

```
[warn] 	module not found: net.ruippeixotog#scala-scraper-config_2.13;2.1.0
[warn] ==== local: tried
[warn]   /Users/patientplatypus/.ivy2/local/net.ruippeixotog/scala-scraper-config_2.13/2.1.0/ivys/ivy.xml
[warn] ==== public: tried
[warn]   https://repo1.maven.org/maven2/net/ruippeixotog/scala-scraper-config_2.13/2.1.0/scala-scraper-config_2.13-2.1.0.pom
[warn] ==== local-preloaded-ivy: tried
[warn]   /Users/patientplatypus/.sbt/preloaded/net.ruippeixotog/scala-scraper-config_2.13/2.1.0/ivys/ivy.xml
[warn] ==== local-preloaded: tried
[warn]   file:////Users/patientplatypus/.sbt/preloaded/net/ruippeixotog/scala-scraper-config_2.13/2.1.0/scala-scraper-config_2.13-2.1.0.pom
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: net.ruippeixotog#scala-scraper-config_2.13;2.1.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		net.ruippeixotog:scala-scraper-config_2.13:2.1.0 (/Users/patientplatypus/Documents/platypusNEST/blog_scala_play/blog/build.sbt#L14-28)
[warn] 		  +- com.example:blog_2.13:1.0-SNAPSHOT
[error] sbt.librarymanagement.ResolveException: unresolved dependency: net.ruippeixotog#scala-scraper-config_2.13;2.1.0: not found
[error] 	at sbt.internal.librarymanagement.IvyActions$.resolveAndRetrieve(IvyActions.scala:332)
[error] 	at sbt.internal.librarymanagement.IvyActions$.$anonfun$updateEither$1(IvyActions.scala:208)
[error] 	at sbt.internal.librarymanagement.IvySbt$Module.$anonfun$withModule$1(Ivy.scala:239)
[error] 	at sbt.internal.librarymanagement.IvySbt.$anonfun$withIvy$1(Ivy.scala:204)
[error] 	at sbt.internal.librarymanagement.IvySbt.sbt$internal$librarymanagement$IvySbt$$action$1(Ivy.scala:70)
[error] 	at sbt.internal.librarymanagement.IvySbt$$anon$3.call(Ivy.scala:77)
[error] 	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:95)
[error] 	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:80)
[error] 	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:99)
[error] 	at xsbt.boot.Using$.withResource(Using.scala:10)
[error] 	at xsbt.boot.Using$.apply(Using.scala:9)
[error] 	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:60)
[error] 	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:50)
[error] 	at xsbt.boot.Locks$.apply0(Locks.scala:31)
[error] 	at xsbt.boot.Locks$.apply(Locks.scala:28)
[error] 	at sbt.internal.librarymanagement.IvySbt.withDefaultLogger(Ivy.scala:77)
[error] 	at sbt.internal.librarymanagement.IvySbt.withIvy(Ivy.scala:199)
[error] 	at sbt.internal.librarymanagement.IvySbt.withIvy(Ivy.scala:196)
[error] 	at sbt.internal.librarymanagement.IvySbt$Module.withModule(Ivy.scala:238)
[error] 	at sbt.internal.librarymanagement.IvyActions$.updateEither(IvyActions.scala:193)
[error] 	at sbt.librarymanagement.ivy.IvyDependencyResolution.update(IvyDependencyResolution.scala:20)
[error] 	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:56)
[error] 	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:45)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:93)
[error] 	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:68)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$19(LibraryManagement.scala:106)
[error] 	at scala.util.control.Exception$Catch.apply(Exception.scala:224)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:106)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:89)
[error] 	at sbt.util.Tracked$.$anonfun$inputChanged$1(Tracked.scala:149)
[error] 	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:120)
[error] 	at sbt.Classpaths$.$anonfun$updateTask$5(Defaults.scala:2561)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:278)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:835)
[error] (update) sbt.librarymanagement.ResolveException: unresolved dependency: net.ruippeixotog#scala-scraper-config_2.13;2.1.0: not found
[error] Total time: 4 s, completed Nov 9, 2019, 2:52:40 PM
```

**Comment by ruippeixotog** [comment]
Thanks for trying out Scala Scraper, @patientplatypus! The problem was on the READMEs and not on the library itself; version 2.2.0 is the first one published for Scala 2.13, so you should use `"net.ruippeixotog" %% "scala-scraper" % "2.2.0"` instead.

I have just updated the README. If it still doesn't work feel free to reopen this!
